content/docs/apis.md

```diff
@@ -2,57 +2,176 @@
+[Raw](#raw) | A lower-level key-value API for interacting directly with individual key-value pairs. | Single key | Your application doesn't require distributed transactions or multi-version concurrency control (MVCC)
+[Transactional](#transactional) | A higher-level key-value API that provides ACID semantics | Multiple keys | Your application requires distributed transactions and/or MVCC
+
+{{< info >}}
+It is **not recommended or supported** to use both the raw and transactional APIs on the same keyspace.
+{{< /info >}}
+There are several clients that connect to TiKV:
+
+* [Rust](https://github.com/tikv/client-rust)
+* [Java](https://github.com/tikv/client-java)
+* [Go](https://github.com/pingcap/tidb/store/tikv)
+
+Below we use the Rust client for some examples, but you should find all clients work similarly.
+
+## Connecting a Client
+
+This guide assumes you are using Rust and have Rust installed through `rustup` or other means.
+
+To start, open the `Cargo.toml` of your project and add the `tikv-client` as a dependency. You'll also need to depend on `futures`.
+
+<!-- TODO: Use crates.to once published -->
+
+```toml
+[dependencies]
+tikv-client = { git = "https://github.com/tikv/client-rust" }
+futures = "0.1"
+```
+
+In your `src/main.rs` you can then import the raw API as well as the functionality of the `Future` trait so you can utilize it.
+
+In this example we used `raw`, but you can also use `transaction`. The process is the same.
+
+```rust
+use tikv_client::{Config, raw::Client}
+use futures::Future;
+```
+
+After building a `Config`, you can then create a `Client`.
+
+```rust
+let config = Config::new(vec![ // Use more than one PD endpoint!
+    "192.168.0.100:2379",
+    "192.168.0.101:2379",
+    "192.168.0.102:2379",
+]).with_security(              // If using TLS.
+    "root.ca",
+    "internal.cert",
+    "internal.key",
+);
+
+let unconnected_client = Client::new(config);
+```
+
+The value returned at this point is a `Future`. It needs to be resolved before we can directly make calls. If your application is syncronous you can call`.wait()` to block the current task until the future is resolved. If your application is asyncronous you might have better ways of dealing with this.
+
+```rust
+let client = unconnected_client.wait()?; // Block and resolve the future.
+```
+
+With a connected client, you'll be able to make requests from TiKV. This client supports both singlular or batch operations.
+
+You can find the full documentation for the client (and all your dependencies) by running:
+
+<!-- TODO: Link to docs.rs page once published -->
+
+```bash
+cargo doc --package tikv-client --open
+```
 
 ## Raw key-value API {#raw}
 
-The raw key-value API is a simple API that enables you to interact with TiKV
+Using a connected `raw::Client` you can perform actions such as basic `put`, `get`, and `delete`:
 
-To try out the raw key-value API:
+```rust
+// `Key` and `Value` wrap around `Vec<u8>` values.
```

Comment:
See below about Value and Key. From the code `key` and `value` look like they are `Vec<u8>`, not wrapped around that.